src/components/Welcome.vue

```diff
@@ -41,7 +41,7 @@ export default {
     /* border: 3px solid #f1f1f1; */
     height:auto;
```

Comment:
Minor detail, but some of these CSS rules can be removed (I think?).  Also, maybe not in this story, but some of the common UI elements, especially between our two stories,  could be moved to a global location.